### PR TITLE
feat: support trace sofa-rpc

### PR DESCRIPTION
**Pull request by gxcsoccer**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 'use strict';
 
 const OpenTracing = require('./lib/opentracing');
-
+const logHTTPClient = require('./lib/trace/http_client');
+const logHTTPServer = require('./lib/trace/http_server');
+const logSofaRpcClient = require('./lib/trace/sofa_rpc_client');
```
Comment (dead-horse) [inline app.js:6]: 这个插件是作为 sofa 这一套一起的还是单独实现 opentracing 相关的东西？

Comment (dead-horse) [reply app.js:6]: 现在看起来是集中式管理，那类似 db / redis 操作的 trace 要做么？

Comment (popomore) [reply app.js:6]: 放到 sofa-tracer-node 吧

Comment (popomore) [reply app.js:6]: 之前没来得及开发

**Comment by popomore** [comment]
https://github.com/eggjs/egg-sofa-tracer

File: app.js (hunk 1 above)
Comment (gxcsoccer) [reply app.js:6]: @popomore 你说的是 [egg-sofa-tracer](https://github.com/eggjs/egg-sofa-tracer) 吧？

**Pull request closed without merging**